index.js

```diff
@@ -435,10 +435,6 @@ export default async function makeHyperFetch ({
       return { status: 404, body: 'Not Found' }
     }
 
-    resHeaders.Link = new URL(path, drive.core.url).href
-
-    resHeaders.ETag = `${entry.seq}`
```

Comment:
This is actually important to help optimize HTTP caching more agressively. The Link header seems to be wrong though

Reply:
I'll address this in the next release

Reply:
Ah, good. Yeah, the Link header was the thing I was trying to fix. Thank you!